project-base/storefront/urql/cacheExchange.ts

```diff
@@ -146,23 +154,21 @@ export const cache = cacheExchange({
                         : undefined;
                 manuallyUpdateCartFragment(cache, newCart);
             },
-            addProductToComparison(result, _args, cache) {
-                invalidateFields(cache, ['comparison']);
-            },
             removeProductFromComparison(result, _args, cache) {
```

Comment:
I would type it as the actual variables object, meaning `args:RemoveProductFromComparisonMutationVariablesApi`. Is there a specific reason why you only typed the UUID?

Reply:
I did not know, I can do that :) Thanks! fixed in 9d97f308cc